### PR TITLE
Nomination Pools: Rename state-toggler to bouncer

### PR DESCRIPTION
**Pull request by rossbulat**
Renames the state toggler role to bouncer.

- [x] Project-wide rename.

Addresses #12747

### CODE REVIEW

File: frame/nomination-pools/src/migration.rs [hunk 1]
```diff
@@ -449,3 +449,96 @@ pub mod v3 {
 		}
 	}
 }
+
+pub mod v4 {
```
Comment (kianenigma) [inline frame/nomination-pools/src/migration.rs:453]: not needed 🙊

**Review by kianenigma — APPROVED** [review]
other then removing migration.

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2417154

**Review by ggwpez — APPROVED** [review]

**Comment by ggwpez** [comment]
bot fmt

**Comment by rossbulat** [comment]
bot rebase

**Comment by rossbulat** [comment]
bot merge

**Pull request merged**